lib/project_printer.rb

```diff
@@ -1,48 +1,51 @@
+require_relative "font_color_and_style"
+
 class ProjectPrinter
-  attr_accessor :stdout
+  attr_accessor :stdout, :colors
 
   def initialize(stdout)
     @stdout = stdout
+    @colors = FontColorAndStyle.new
   end
 
   def projects(project)
-    stdout.puts("  #{project.keys.first}\n")
+    stdout.puts("  #{project.keys.first}")
   end
 
   def deleting_a_project(project_name)
-    stdout.puts "\e[38;5;40mDeleting project:\e[0m '#{project_name.strip}'\n\n"
+    stdout.puts colors.success("Deleting project: '#{project_name.strip}'")
   end
 
   def does_not_exist(project_name)
-    stdout.puts "\e[40;38;5;214mProject doesn't exist:\e[0m '#{project_name.strip}'\n\n"
+    stdout.puts colors.alert("Project doesn't exist: '#{project_name.strip}'")
```

Comment:
I guess this is used as an alert too! You can use a second duplicate method in the formatter class to disambiguate alerts and prompts. Then in the future if you wanted the prompts to look less alert-y it would be pretty trivial. But also it will read more semantically.